tools/ja-rule/ja-rule.cpp

```diff
@@ -170,9 +209,43 @@ class MessageHandler : public MessageHandlerInterface {
   void PrintResponse(const Message& message) {
     OLA_INFO << "RC (" << static_cast<int>(message.return_code)
              << "): payload_size: " << message.payload_size;
-    if (message.payload && message.payload_size) {
-      ola::strings::FormatData(&std::cout, message.payload,
-                               message.payload_size);
+
+    if (message.payload_size && message.payload) {
+      rdm_response_code response_code;
+      // Ignore the start code.
+      auto_ptr<RDMResponse> response(RDMResponse::InflateFromData(
+          message.payload + 1, message.payload_size - 1, &response_code));
+
+      if (!response.get()) {
+        OLA_WARN << "Failed to inflate RDM response";
+        if (message.payload == NULL || message.payload_size == 0) {
+          ola::strings::FormatData(&std::cout, message.payload,
+                                   message.payload_size);
+        }
+      } else {
+        OLA_INFO << *response;
+      }
+    }
+  }
+
+  void PrintTime(const Message& message, TimeFormat format) {
+    uint16_t time = 0;
```

Comment:
Is it worth using an unreserved word? It's certainly confusing GitHub's formatting.